### DISCUSSION TITLE
Multi-user project : pictures sharing

### DISCUSSION THREAD

**Discussion by dukytony** [opengisch/QField#3007] [Q&A]
Hello everybody,

I'm currently working on a massive GIS for an archaeological mission and looking for a picture cloud/sharing solution.

The picture inventory is an online postgresql table accessible via QField running on 4 android tablets with 4G connection.

My goal is the following : 

- on field pictures will be taken using qfield and stored on sd card storage using a similar name folder on each tablets.
- every evening all pictures taken in the different tablets should be synchronized automatically using wifi on a shared drive so that every picture taken with every tablets should be available locally on every tablets on sd cards the next day.

So I was thinking about creating local wifi network with a router and hook up a NAS server as shared drive for this purpose. Then using a syncronizing app to upload/download pictures to and from the NAS every evening.

At first i was thinking of doing all of it online through ftp/http but I doubt we will have a good enough internet access for this at our hotel (remote place) and it would consume too much data to do it through 4G.

Any of you guys had to do similar projects ? any better ideas on how to do this more efficiently?

Thank you very much to you all!

**Comment by lucienicolier** [comment]
Hi @dukytony,

We will be pleased to help you, I suggest you contact info@opengis.ch for a support offer.

Best regards
